OsmIntegrator/Database/DataInitialization/DataInitializer.cs

```diff
@@ -27,116 +27,110 @@ public DataInitializer(IConfiguration configuration)
         public void Initialize(ApplicationDbContext db)
         {
             using (var transaction = db.Database.BeginTransaction())
+            {
+                try
                 {
-                    try
+                    if (db.Stops.Count() > 0)
                     {
-                        if (db.Stops.Count() > 0)
-                        {
-                            return;
-                        }
+                        return;
+                    }
+
+                    Guid editorRoleId = Guid.Parse("ff6cce18-706d-448c-89de-56212f1722ef");
```

Comment:
czemu nie Guid.NewGuid(); ?

Reply:
Bo to są dane inicjowane w bazie i chciałbym, żeby identyfikator był za każdym razem taki sam.